src/main/java/mate.academy/Main.java

```diff
@@ -8,10 +8,14 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
-        List<Product> products = productService.getAllFromFile("products.txt");
-        products.forEach(System.out::println);
+        try {
+            ProductService productService =
+                    (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Ensure that the `Injector` class is correctly implemented and configured to provide instances of `ProductService`. If the `Injector` is not set up properly, this line will throw a `RuntimeException`.